demo/zip_two_files/run.py

```diff
@@ -13,7 +13,7 @@ def zip_two_files(file1, file2):
 
 demo = gr.Interface(
     zip_two_files,
-    ["file", "file"],
+    [gr.File(file_count="multiple"), gr.File(file_count="multiple")],
```

Comment:
Why did we change both of the input components here to accept multiple files each? We should either have a single component that can accept multiple files (and change the name of this demo) or just change it back to two file components that each accept a single file

Reply:
Pushed this by accident. Changed this for testing purposes